web/portal/brand/src/translations/en.json

```diff
@@ -13,8 +13,8 @@
   "Add QR Code to user portal to provision GS Wave mobile softphone": "Add QR Code to user portal to provision GS Wave mobile softphone",
   "Administrator access privilege_one": "Administrator access privilege",
   "Administrator access privilege_other": "Administrator access privileges",
-  "Administrator_one": "Administrator",
-  "Administrator_other": "Administrators",
+  "Administrator_one": "Administrator of client",
```

Comment:
Changing just the translation will make imposible to show just "Administrator" in the future